index.html

```diff
@@ -12,7 +12,13 @@
     </header>
     <main>
       <table id="main-table">
+        <thead id="table-head">
```

Comment:
The HTML looks fantastic. Thanks for working on the white space and indentation!